scipy/conftest.py

```diff
@@ -180,23 +228,3 @@ def check_fpu_mode(request):
 SCIPY_HYPOTHESIS_PROFILE = os.environ.get("SCIPY_HYPOTHESIS_PROFILE",
                                           "deterministic")
 hypothesis.settings.load_profile(SCIPY_HYPOTHESIS_PROFILE)
-
-
-def skip_if_array_api_backend(backend):
```

Comment:
this function is moved above the `hypothesis` stuff from #18927 so that the array API stuff is not split on either side.